src/components/admin-dashboard/TransactionTable.js

```diff
@@ -22,7 +22,7 @@ const useStyles = makeStyles(theme =>({
 
     '& span': {
         fontWeight: '300',
-        padding: '4.6px 0',
+        padding: '10px 0',
```

Comment:
Try and replace these pixel values with MUI's theme.spacing( ) function @Godstrump